package.json

```diff
@@ -9,31 +9,24 @@
     "build:cjs": "lerna exec --parallel -- babel src -d dist --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:esm": "BABEL_TYPE=esm lerna exec --parallel -- babel src -d esm --configFile babel.config.js --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:storybook": "build-storybook -o storybook_dist",
-    "ci:packages": "npm ci && lerna exec -- npm ci",
-    "changeScanner": "node scripts/changeScanner",
-    "create:package": "yo .yeoman",
     "bumpPackages": "node scripts/bumpPackages/cli",
     "bumpChangelogs": "node scripts/bumpChangelogs/cli",
-    "regeneratePackageLocks": "node scripts/regeneratePackageLocks",
+    "bumpPackages": "node scripts/bumpPackages/cli",
```

Comment:
This is now in here twice? line 12 is a duplicate thats out of alphabetical order